content/docs/services/intro.md

```diff
@@ -33,7 +33,7 @@ To list all the managed services and plans available to a given space, you run `
 
 *Note that when you extend the cloud.gov marketplace with your own broker, the cloud.gov team cannot vouch for the security or compliace of the brokered services. You will need to document and authorize your own brokered services in accordance with  your agency's compliance requirements.*
 
-We've supplied a [tutorial to help you understand how to operate your own broker](https://github.com/18F/cf-byo-broker), including instructions for deploying some sample brokers into cloud.gov. Once you've reviewed this tutorial, you may want to investigate some of the [community-supported broker add-ons for Cloud Foundry](https://github.com/cloudfoundry-community?q=broker) such as the [app-autoscaler](https://github.com/cloudfoundry-incubator/app-autoscaler).
+This [tutorial will help you understand how to operate your own broker](https://github.com/18F/cf-byo-broker), and includes instructions for deploying some sample brokers into cloud.gov. Once you've reviewed this tutorial, you may want to investigate some of the [community-supported broker add-ons for Cloud Foundry](https://github.com/cloudfoundry-community?q=broker) such as the [app-autoscaler](https://github.com/cloudfoundry-incubator/app-autoscaler).
```

Comment:
Also not a blocker, but I prefer the framing "This tutorial has instructions for how to x" instead of "This tutorial will help you understand how to x" because I can't promise that it'll help a person understand something (despite our best efforts).

Reply:
I'll make a separate PR to address this point; I wanted to get this merged and live ASAP so that Amirah can copy-pasta from the rendered output into GovDelivery and send the bulletin part, which shouldn't wait for that update.